src/intervals/real_interface.jl

```diff
@@ -0,0 +1,53 @@
+#= This file contains the function that must be defined on Intervals
+    so that they behave like Real in julia.
+=#
+
+real(a::Interval) = a
+
+zero(a::F) where {T<:Real, F<:Interval{T}} = F(zero(T))
+zero(::Type{F}) where {T<:Real, F<:Interval{T}} = F(zero(T))
+
+one(a::F) where {T<:Real, F<:Interval{T}} = F(one(T))
+one(::Type{F}) where {T<:Real, F<:Interval{T}} = F(one(T))
+
+typemin(::Type{F}) where {T<:Real, F<:Interval{T}} = F(typemin(T), nextfloat(typemin(T)))
+typemax(::Type{F}) where {T<:Real, F<:Interval{T}} = F(prevfloat(typemax(T)), typemax(T))
+typemin(::Type{F}) where {T<:Integer, F<:Interval{T}} = F(typemin(T))
+typemax(::Type{F}) where {T<:Integer, F<:Interval{T}} = F(typemax(T))
```

Comment:
Naive question: Do we need the last two methods?

Reply:
Yes, `prevfloat` and `nextfloat` error on integers.